README.md

```diff
@@ -244,11 +244,22 @@ const article = new Schema('articles');
 // You can use a custom id attribute
 const article = new Schema('articles', { idAttribute: 'slug' });
 
-// Or you can specify a function to infer it
-function generateSlug(entity) { /* ... */ }
+// Or you can specify a function to infer it.
+function generateSlug(entity, collectionKey) { /* ... */ }
```

Comment:
`key` or `entityKey` would make more sense here.

Even better, the input isn't `entity` (yet), it's the object provided before normalization, so I almost feel like `value`, `key` would make more sense.